ouroboros-consensus/src/unstable-consensus-testlib/Test/Util/Serialisation/Roundtrip.hs

```diff
@@ -148,11 +213,62 @@ roundtrip_all
   => CodecConfig blk
   -> (forall a. NestedCtxt_ blk Header a -> Dict (Eq a, Show a))
   -> TestTree
-roundtrip_all ccfg dictNestedHdr =
+roundtrip_all = roundtrip_all_skipping (const CheckCBORValidity)
+
+-- | All roundtrip tests, skipping the specified CBOR validity tests.
+--
+--
+-- TODO: the exclusion rule should only be considered for blocks __before__ Conway!
```

Comment:
Would it make sense to address this as a separate issue?

Reply:
AFAICT, no Conway blocks are excluded at all, is this comment stale (ie a leftover from the `FlatTerm` roundtripping)?

Reply:
Oh, what I meant is that we are using this to skip checks for legacy encoders *before* Conway. However, we should have a way to enable `roundtrip_SerialiseNodeToClient` at and after Conway. So I was thinking about changing the type of the filter function to something like:

```haskell
(a, TestName) -> ShouldCheckCBORValidity
```